src/Resources/ResourceManager/Implementation/Policy/NewAzurePolicyAssignment.cs

```diff
@@ -75,6 +75,7 @@ public class NewAzurePolicyAssignmentCmdlet : PolicyCmdletBase, IDynamicParamete
         /// <summary>
         /// Gets or sets the policy assignment policy definition parameter.
         /// </summary>
+        [CmdletParameterBreakingChange("PolicyDefinition", OldParamaterType = typeof(PSObject), NewParameterTypeName = "PsPolicyDefinition")]
```

Comment:
these are the only places that the cmdlets take in policy entities?

Reply:
Yes, it's the only place the current ones do, hence the only break. The new ones take policy entities in new InputObject parameters.